products/pdt/pdtparse/src/events.rs

```diff
@@ -19,6 +19,7 @@ pub enum ZRC2Event {
 pub struct MintedEvent {
     txn_id: String,
     block: i64,
+    #[allow(dead_code)]
```

Comment:
Added to shut the compiler again. I thought it was better than the alternative of prepending `_`